app/views/content/international-qualified-teacher-status/_after-accordion.md

```diff
@@ -4,12 +4,21 @@ Learn more about [iQTS policy](https://www.gov.uk/government/publications/intern
 
 Learn more about [other routes to qualified teacher status (QTS) and teaching in England](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk#apply-to-the-teaching-regulation-agency-tra).
 
-[Come to England to teach if you're a teacher from outside the UK](/come-to-england-to-teach-if-you-are-a-teacher-from-outside-the-uk) gives information and advice to international teachers who’d like to teach in England.
+[Find help and guidance for international teachers who'd like to teach in England.](/come-to-england-to-teach-if-you-are-a-teacher-from-outside-the-uk)
 
 If you already live in the UK, you should consider [applying for teacher training leading to QTS in England](/steps-to-become-a-teacher). 
 
-This guidance relates to teaching in England. Different rules apply to teaching in [Wales](https://www.ewc.wales/site/index.php/en/registration/practitioners-trained-outside-wales.html), [Scotland](https://teachinscotland.scot/become-a-teacher/qualified-outside-scotland/) and [Northern Ireland](https://gtcni.org.uk/registration/getting-registered).
+This guidance relates to teaching in England. Different rules apply to teaching in [Wales,](https://www.ewc.wales/site/index.php/en/registration/practitioners-trained-outside-wales.html) [Scotland](https://teachinscotland.scot/become-a-teacher/qualified-outside-scotland/) and [Northern Ireland](https://gtcni.org.uk/registration/getting-registered).
```

Comment:
This section was flagged in our DAC accessibility audit - one-word links with no context are not good for screen reader users. To address in a separate ticket/PR rather than hold up this one

Reply:
I've changed this but you can see how it makes writing these lists of regional differences slihgtly awkward.